docs/package.json

```
@@ -30,6 +30,7 @@
 		"typedoc-vitepress-theme": "1.0.0-next.2",
 		"typescript": "5.0.4",
 		"vitepress": "1.0.0-alpha.75",
+		"vitepress-plugin-tabs": "^0.3.0",
```

Lets fix this version 

```suggestion
		"vitepress-plugin-tabs": "0.3.0",
```